From 21404d1a39486112eb1f46063b9597ecfd2fde35 Mon Sep 17 00:00:00 2001
From: yxj <yxj@rock-chips.com>
Date: Wed, 17 Sep 2014 14:24:18 +0800
Subject: [PATCH] dts:rk3126-86v: fb: add node rockchip,uboot-logo-on

	set this node to 1 will enable boot logo display in uboot
---
 arch/arm/boot/dts/rk3126-86v.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3126-86v.dts b/arch/arm/boot/dts/rk3126-86v.dts
index 98cedd303f62..e9c1e4a4eef7 100644
--- a/arch/arm/boot/dts/rk3126-86v.dts
+++ b/arch/arm/boot/dts/rk3126-86v.dts
@@ -171,7 +171,8 @@
 };
 
 &fb {
-        rockchip,disp-mode = <NO_DUAL>;
+	rockchip,disp-mode = <NO_DUAL>;
+	rockchip,uboot-logo-on = <0>;
 };
 
 &rk_screen {
-- 
2.35.3

